requirements.txt

```diff
@@ -1 +1,2 @@
 -e git+https://github.com/f5devcentral/f5-ctlr-agent.git@671a0defbea9e7def753adb685cbd9230754f131#egg=f5-ctlr-agent
+-e git+https://github.com/f5devcentral/f5-cccl.git@cfaaa7713fddf9fe28400317548d25b104d3b865#egg=f5-cccl
```

Comment:
At one point, the CC team intentionally removed direct dependency between the controller and cccl, and instead added a direct dependency only on f5-ctlr-agent. I expect you have to do this since the process-dependency-links option is no longer supported. Perhaps, it is worth to see how multi-level dependencies can be leveraged without calling them out at the top level.

Reply:
We will create a backlog item to evaluate a permanent solution for this issue.